packages/api/src/external/fhir/document/index.ts

```diff
@@ -316,20 +325,43 @@ function convertCWIdentifierToFHIR(
 }
 
 /**
- * Extracts the author organization from the contained resources.
+ * Get the authors from content that match the contained resources.
  *
+ * @param content CW DocumentContent containing the doc ref's data coming from CW.
  * @param contained FHIR-compliant resources built from CW resources.
- * @returns FHIR Reference to the author organization; otherwise undefined if a valid author organization cannot be determined.
+ * @returns FHIR References to be used as authors to the document reference.
  */
-function extractAuthorOrganization(contained: Resource[]): Reference<FhirOrganization> | undefined {
-  const org = contained.find(r => r.resourceType === "Organization");
-  if (org) {
-    return {
-      // https://www.hl7.org/fhir/R4/references-definitions.html#Reference.reference
-      reference: `Organization/${org.id}`,
-      type: "Organization",
-    };
-  }
+export function getAuthors(
+  content: DocumentContent,
+  contained: Resource[],
+  docId: string
+): Reference<FhirOrganization | Practitioner>[] {
```

Comment:
won't the return be any one of the types in `authorTypes`?

Reply:
I'll check this out, good catch. I think the type can't be determined, but I'll double check and try to make it match.

Reply:
updated